app/src/main/java/ar/com/wolox/android/training/model/NewsItem.kt

```diff
@@ -1,14 +1,34 @@
 package ar.com.wolox.android.training.model
 
 import com.google.gson.annotations.SerializedName
+import java.text.SimpleDateFormat
 import java.util.Date
+import java.util.Locale
 
-data class NewsItem(val title: String, val text: String) {
-    var picture: String = ""
-    var userLike: Boolean = false
-    var date: Date = Date()
-    var likes: List<Int> = arrayListOf()
-
+data class NewsItem(
+    val title: String,
+    val text: String,
+    var picture: String,
+    var userLike: Boolean,
+    var date: Date,
+    val likes: List<Int>,
     @SerializedName("createdAt")
-    var createdAt: String = ""
+    var createdAt: String
+) {
+    fun updateDate() {
```

Comment:
I meant something different. What I wanted is something like
```
val createdAtFormatted = SimpleDateFormat(DATE_FORMAT, Locale.getDefault()).parse(createdAt)
```

I think since it comes from a JSON maybe you will need to make that a method or a lazy.